parse_model_2.ipynb

```diff
@@ -0,0 +1,581 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from pythonscript.rectangle_parser import run_parser\n",
+    "image_path = '/Users/elifnazduman/Desktop/E.D ÖZEL/python/uml-parser/uml.png'\n",
```

Comment:
Can you point this to the image in repository please, otherwise I won't be able to access this